docs/community/governance.md

```diff
@@ -52,13 +52,18 @@ have shown they can be trusted to maintain napari with care. Becoming a
 core developer allows contributors to merge approved pull requests, cast votes
 for and against merging a pull-request, and be involved in deciding major
 changes to the API, and thereby more easily carry on with their project related
-activities. Core developers appear as organization members on the napari
-[GitHub organization](https://github.com/orgs/napari/people) and are on our
-[@napari/core-devs](https://github.com/orgs/napari/teams/core-devs) GitHub team. Core
-developers are expected to review code contributions while adhering to the
+activities.
+
+Core developers are expected to review code contributions while adhering to the
 [core developer guide](core-dev-guide). New core developers can be nominated
 by any existing core developer, and for details on that process see our core
-developer guide. For a full list of core developers see our [About Us](team) page.
+developer guide. Core developers can choose to step down from their role and 
+become "emeritus" core developers if they are no longer involved with the project,
+or feel they do not have sufficient time to dedicate to their role. Emeritus core
+developers can request or be invited to become active core developers at
+a later date and with consensus from currently active core developers. 
+
+For a full list of core developers see our [About Us](team) page.
```

Comment:
either "about the project and team" or just "about"? "About Us" doesn't exist as a string anywhere there.

Reply:
> "About Us" doesn't exist as a string anywhere there.

Well maybe it should :laughing: 

Updated in latest commit.